### PR TITLE
Change to environment specific image building and deploy.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Deploy goodness

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -170,7 +237,7 @@ workflows:
               only:
                 - /.*/
             tags:
-              only:
+              ignore:
                 - /v.*/
```
Comment (jmartin-sul) [inline .circleci/config.yml:241]: should this be `/.*/`?  i assume we don't want to run `lint` for any tagging?  or do we want to run it for the non `/v.*/` tags?

File: .circleci/config.yml [hunk 2]
```diff
@@ -180,15 +247,15 @@ workflows:
...
```
Comment (jmartin-sul) [inline .circleci/config.yml:251]: same question about ignoring all here

File: .circleci/config.yml [hunk 3]
```diff
@@ -180,15 +247,15 @@ workflows:
...
       - integration:
           filters:
             branches:
               only:
                 - /.*/
             tags:
-              only:
+              ignore:
                 - /v.*/
```
Comment (jmartin-sul) [inline .circleci/config.yml:259]: same question about ignoring all here

**Review by jmartin-sul — APPROVED** [review]
merging despite the nitpicks, since they're pretty minor even if technically correct.

File: .circleci/config.yml (hunk 1 above)
Comment (justinlittman) [reply .circleci/config.yml:241]: Yeah, this change was intentional. No need to run linting for version tags.

**Pull request merged**